src/nacl/bindings/crypto_box.py

```diff
@@ -304,10 +304,11 @@ def crypto_box_seal_open(ciphertext, pk, sk):
     _clen = len(ciphertext)
     _mlen = _clen - crypto_box_SEALBYTES
 
```

Comment:
I'm unsure about adding a explicit check to raise ValueError when `_mlen < 0`.

Reply:
to clarify, if we would decide add a explicit size check, IMO the raised exception should be a `nacl.exception.CryptoError` subclass; both `nacl.exception.TypeError`  and `nacl.exception.ValueError` could make sense.

Reply:
If we do an explicit check why not just do `_mlen <= 0` and then we don't need to do the `max()` code below. Or is it possible to seal an empty byte string such that when you call crypto_box_seal_open you would expect an empty byte string response?

Reply:
While I didn't add a explicit test for that case, I verified sealedbox box can ship **and verify** a zero-length message while preparing the PR. If you think the test can be useful, I can either add it in this PR, or in a follow-up one.

Reply:
Still, if we add a explicit check for `_mlen < 0`, we can skip the second `max`.

Reply:
Let's add the negative test so we can remove that second max